extensions/amp-subscriptions/0.1/test/test-platform-store.js

```diff
@@ -1008,4 +1012,23 @@ describes.realWin('Platform store', {}, (env) => {
       expect(platform.reset).to.be.calledOnce;
     });
   });
+
+  describe('resetPlatformStore', () => {
+    it('the entitlement callback added through public API should persist even after the platform store gets reset', async () => {
+      const isCallbackCalled = new Deferred();
+      platformStore.addOnEntitlementResolvedCallback(() => {
```

Comment:
Please use a spy or a stub so you can check how many times it's been called, and with which arguments.

Reply:
This method won't be called when resetting the platform store. I am just using this method to add a callback and to make sure that callback would still persist after resetting the platform store. Or you mean adding a test case for addOnEntitlementResolvedCallback() itself?

Reply:
Added the testing for addOnEntitlementResolvedCallback inside the same test case.

Reply:
Ah neat! Sorry missed the comment. Updated with the suggested solution.